app/src/main/java/com/github/sdp/mediato/api/themoviedb/models/TMDBMovie.java

```diff
@@ -0,0 +1,36 @@
+package com.github.sdp.mediato.api.themoviedb.models;
+
+import java.util.List;
+
+public class TMDBMovie {
+    private String poster_path;
```

Comment:
Sam remark as in PagedResult, final attributes

Reply:
Same answer